core/frontend/src/components/common/DevicePathHelper.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <v-tooltip>
+    <template #activator="{ on, attrs }">
+      <v-icon
+        v-if="board_connector"
+        class="ml-4"
+        v-bind="attrs"
+        v-on="on"
+      >
+        mdi-information
+      </v-icon>
+    </template>
+    <img :src="board_image">
+    <div :class="circle_class" />
+  </v-tooltip>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+import system_information from '@/store/system-information'
+import { Dictionary } from '@/types/common'
+import { SerialPortInfo } from '@/types/system-information/serial'
+
+enum BoardType {
+  Rpi4B = 'Rpi4B',
+  Rpi3B = 'Rpi3B',
+  Navigator = 'Navigator',
+  Unknown = 'Unknown'
+}
+
+const connector_map: Dictionary<string> = {
+  '/dev/ttyS0': 'serial1',
+  '/dev/ttyAMA1': 'serial3',
+  '/dev/ttyAMA2': 'serial4',
+  '/dev/ttyAMA3': 'serial5',
+  '/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.3:1.0-port0': 'usb-top-left-pi4',
+  '/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.4:1.0-port0': 'usb-bottom-left-pi4',
+  '/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.1:1.0-port0': 'usb-top-right-pi4',
+  '/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.2:1.0-port0': 'usb-bottom-right-pi4',
+  '/dev/serial/by-path/platform-3f980000.usb-usb-0:1.5:1.0-port0': 'usb-bottom-right-pi3', // RPI3
+  '/dev/serial/by-path/platform-3f980000.usb-usb-0:1.4:1.0-port0': 'usb-top-right-pi3', // RPI3
+  '/dev/serial/by-path/platform-3f980000.usb-usb-0:1.3:1.0-port0': 'usb-bottom-left-pi3', // RPI3
+  '/dev/serial/by-path/platform-3f980000.usb-usb-0:1.2:1.0-port0': 'usb-top-left-pi3', // RPI3
+}
```

Comment:
I am not sure, if the maps don't work, it should just not show anything (not even the (i) iicon)

Reply:
I believe this whole by-path idea will work when it refers to the motherboard's native USB, but I don't think this will take care when someone uses some USB hub, we should test it.

Reply:
Well done @Williangalvani ! So, with a regex we maybe could have an indication of where the hub is connected to, clearly saying that we are talking about the hub, not the device, but it is way beyond what we want for now, so I leave it here just as something to be improved next time, maybe.